cpu/native/async_read.c

```diff
@@ -42,6 +42,8 @@ static void _async_io_isr(void) {
 
     int max_fd = 0;
 
+    struct timeval timeout = { .tv_usec = 1 };
```

Comment:
Confirmed "To effect a poll, the timeout argument should be non-nil, pointing to a zero-valued timeval structure." Amended.